proto/v1/vulnerability.proto

```diff
@@ -108,6 +109,10 @@ message VulnerabilityNote {
     // Whether this detail is obsolete. Occurrences are expected not to point to
     // obsolete details.
     bool is_obsolete = 11;
+
+    // The time this information was last changed at the source. This is an
+    // upstream timestamp from the underlying information source.
```

Comment:
Let's include examples of the information source at the end. Ditto for the below.

Reply:
Added an example of the Ubuntu Security Tracker